pages/documents/DeveloperTools/FaaS/Developing/toolbelt.md

```diff
@@ -192,6 +192,74 @@ This method retrieves a conversation from the [Messaging Interactions API](https
   .catch(err => //TODO: React to error);
 ```
 
+#### Scan Conversation For Keywords
+
+This method scans a conversation that has been retrieved with `getConversationById()` (s. method above) for messages containing certain keywords. Those keywords can be freely determined.
```

Comment:
Please add information that keywords are case insensitive.

Reply:
implemented